testsuite/MDAnalysisTests/topology/test_lammpsdata.py

```diff
@@ -178,6 +179,21 @@ def test_traj(self, filename):
                                [12.6588821411, 51.4160842896, 20.5548400879]],
                               dtype=np.float32))
 
+class TestLammpsDataPairIJ(LammpsBase):
```

Comment:
Since you're subclassing `LammpsBase` but your expected attributes don't include everything (in this case charges), could you should just redefine `expected_attrs` in the subclass to match the attributes contained in this new file?

Reply:
@IAlibay Thank you for pointing everything out. I've addressed the issues.